package.json

```diff
@@ -48,7 +48,7 @@
     "devDependencies": {
         "@commitlint/cli": "16.2.4",
         "@commitlint/config-conventional": "16.2.4",
-        "@coveo/semantic-monorepo-tools": "0.4.0",
```

Comment:
A lot of other stuff changed between 0.4.0 and 1.1.0, I don't think the lib returns the process anymore. See the changelog for more info

https://github.com/coveo/semantic-monorepo-tools/blob/main/CHANGELOG.md

Reply:
IIRC everything is now async only